sqlite3.go

```diff
@@ -297,6 +297,51 @@ const (
 	/*SQLITE_RECURSIVE           = C.SQLITE_RECURSIVE*/
 )
 
+// Standard File Control Opcodes
+// See: https://www.sqlite.org/c3ref/c_fcntl_begin_atomic_write.html
+const (
+	SQLITE_FCNTL_LOCKSTATE             = 1
```

Comment:
```
SQLITE_FCNTL_LOCKSTATE = int(C.SQLITE_FCNTL_LOCKSTATE)
```

Reply:
Fixed 👍

Reply:
@mattn Good call. I updated it and push up a new commit. :+1